### DISCUSSION TITLE
Data Entered into Pages Does Not Save

### DISCUSSION THREAD

**Discussion by Imb12345** [outline/outline#7002] [Self Hosting]
Hello, I am running Outline via the “outline-docker-compose” repo, and am encountering a problem with pages. When I enter text into a page, it remains for a time, but I will come back several days later for the text to be gone. No Docker container restarts or anything like that. Page content simply vanishes. Please advise. Thank you.

**To Reproduce**
1. Create a page within Outline
2. Enter some text into the page
3. Come back several days later to see text has vanished and is no longer present

**Expected behavior**
Pages are expected to retain information, including text, images, and links entered into them.

**Outline (please complete the following information):**
- Install: outline-docker-compose repo: https://github.com/vicalloy/outline-docker-compose

**Comment by tommoor** [comment]
It sounds like it isn't persisting to the server, I would check that both websockets are connecting.
